openapi.yaml

```diff
@@ -0,0 +1,425 @@
+openapi: 3.0.1
+
+info:
+  title: Zero Wallet Backend server
+  version: 0.1.0
+  description: handles requests for ZeroWallet frontend sdk
+
+servers:
+  - url: https://api-zero-wallet.questbook.app
+    description: Production server
+  - url: http://localhost:3000
+    description: Local server
+
+security:
+  - BearerAuth: []
+paths:
+
+  /auth/authorize:
+    post:
+      operationId: authorize
+      summary: add the given user as authorized to the database
+      tags:
+        - Auth
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AuthorizeRequest'
+      responses:
+        '200':
+          $ref: '#/components/responses/AuthorizeResponse'
+        '400':
+          $ref: '#/components/responses/ErrorResponse'
+        '500':
+          $ref: '#/components/responses/ErrorResponse'
+
+  /auth/getNonce:
```

Comment:
best to stick to a single type of casing on routes, either snake or camel

Reply:
Fixed.